libvast/src/system/importer.cpp

```diff
@@ -126,6 +127,18 @@ void importer_state::send_report() {
     auto node_throughput = measurement{elapsed, measurement_.events};
     auto r = performance_report{
       {{"importer"s, measurement_}, {"node_throughput"s, node_throughput}}};
+#if VAST_LOG_LEVEL >= VAST_LOG_LEVEL_VERBOSE
+    auto beat = [&](performance_sample s) {
```

Comment:
```suggestion
    auto beat = [&](const auto& sample) {
```

We shouldn't needlessly copy here.